mne/stats/cluster_level.py

```diff
@@ -1978,13 +1993,20 @@ def plot_cluster(self, condition_labels: dict):
         condition_labels : dict
             Dictionary with condition labels as keys and evoked objects as values.
         """
+        # define colorblind friendly colors
+        colorblind_palette = ["#4daf4a", "#984ea3"]
```

Comment:
we should in general not hard-code colors in the source code. I suggest exposing a `colors=None` keyword arg, then passing that through to `plot_compare_evokeds` so the user can control the color.

This raises the question of what *other* params of plot_compare_evokeds we might want to expose here... `linestyles`, `styles`, `cmap`, ???